MetaMask/metamask-extension
# Fix #1203 - Add iframe for analytics during onboarding when participating in MetaMetrics

## Explanation

* Fixes [#12345](https://github.com/MetaMask/MetaMask-planning/issues/1203)

I injected this iframe at the "Create Password" step of onboarding because it's the first common screen between Importing a Wallet and Creating a Wallet, as well as after the user has opted in or out of MetaMetrics.

## Manual Testing Steps

1.  Create a new wallet, opt into metametrics
2.  On "Create a password screen", inspect the elements to see the iframe
3.  Create another new wallet, opt out metametrics
4. On "Create a password screen", inspect the elements to *not* see the iframe
5. (Repeat these steps during import process)

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
